daffodil-test/src/test/scala/org/apache/daffodil/section00/general/TestTextBidi.scala

```diff
@@ -1,42 +1,42 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.daffodil.section00.general
-
-import org.apache.daffodil.tdml.Runner
-import org.junit.{ AfterClass, Test }
-
-object TestTextBidi {
-  val testDir = "/org/apache/daffodil/section00/general/"
-  val runner = Runner(testDir, "testBidi.tdml")
-
-  @AfterClass def shutDown: Unit = {
-    runner.reset
-  }
-}
-
-/*
- * This test addresses reported issue DAFFODIL-2322
- * dfdl:textBidi property errors when used in attribute form
- */
-class TestTextBidi {
-
-  import TestTextBidi._
-
-  @Test def test_textBidi(): Unit = { runner.runOneTest("textBidi1") }
-
-}
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.daffodil.section00.general
+
+import org.apache.daffodil.tdml.Runner
+import org.junit.{ AfterClass, Test }
+
+object TestTextBidi {
+  val testDir = "/org/apache/daffodil/section00/general/"
+  val runner = Runner(testDir, "testTextBidi.tdml")
+
+  @AfterClass def shutDown: Unit = {
+    runner.reset
+  }
+}
+
+/*
+ * This test addresses reported issue DAFFODIL-2322
+ * dfdl:textBidi property errors when used in attribute form
+ */
+class TestTextBidi {
+
+  import TestTextBidi._
+
+  @Test def test_textBidi(): Unit = { runner.runOneTest("test_textBidiNo") }

```

Comment:
The convention we use for naming tests is the name of the JUnit test should be thes ame as the name of the tdml test, but with a ``test_`` prefix. So this tdml test name should not have ``test_`` and the JUnit name needs to be ``test_testBidiNo`` (note the "No" at the end).